pkg/network/deviceinfo/sriov.go

```diff
@@ -57,45 +55,27 @@ func CreateNetworkPCIAnnotationValue(networks []v1.Network, interfaces []v1.Inte
 
 func mapNetworkNameToPCIAddress(networks []v1.Network, interfaces []v1.Interface,
 	networkStatusAnnotationValue string) (map[string]string, error) {
-	multusInterfaceNameToNetworkStatusMap, err := mapMultusInterfaceNameToNetworkStatus(networkStatusAnnotationValue)
+	networkNameToDeviceInfo, err := mapNetworkNameToDeviceInfo(networks, networkStatusAnnotationValue,
+		vmispec.FilterSRIOVInterfaces(interfaces))
```

Comment:
format: If it is too long, please use 
```go
foo(
  a,
  b,
)
```

When breaking the line in the middle of the arguments, it because hard to understand what is going on. We need a format tool that can do this for us.

Reply:
done.
We indeed need a tool doing it for us.